lib/helpers.js

```diff
@@ -209,8 +209,28 @@ function buildStartCmd (startAppOptions, apiLevel) {
   return cmd;
 }
 
+async function getSdkToolsVersion () {
+  const androidHome = process.env.ANDROID_HOME;
+  if (!androidHome) {
+    log.errorAndThrow('ANDROID_HOME environment varibale is expected to be set');
+  }
+  const propertiesPath = path.resolve(androidHome, 'tools', 'source.properties');
+  if (!await fs.exists(propertiesPath)) {
+    log.warn(`Cannot find ${propertiesPath} file to read SDK version from`);
+    return;
+  }
+  const propertiesContent = await fs.readFile(propertiesPath, 'utf8');
+  const versionMatcher = new RegExp(/Pkg\.Revision=(\d+)\.?(\d+)?\.?(\d+)?/);
+  const match = versionMatcher.exec(propertiesContent);
+  if (match) return {
```

Comment:
Thanks for the hint. I just follow [this](https://github.com/appium/appium/blob/master/docs/en/contributing-to-appium/style-guide-2.0.md) document and it's stated there, that one may use return without brackets for short-circuiting. I though this rule might be applied here as well.